api/app/controllers/spree/api/v2/platform/products_controller.rb

```diff
@@ -5,6 +5,23 @@ module Platform
         class ProductsController < ResourceController
           include ::Spree::Api::V2::ProductListIncludes
 
+          def translate
+            return render_error_payload('No automated translations provider configured') unless automated_translations_service.enabled?
```

Comment:
Shouldn't we manage this error via i18n?

Reply:
We should 🙂